pontoon/base/models.py

```diff
@@ -213,6 +219,14 @@ class Meta:
             ('can_manage_locale', 'Can manage locale')
         )
 
+    def get_plural_index(self, cldr_plural):
+        """Returns plural index for given cldr name."""
+        cldr_id = Locale.cldr_plural_to_id(cldr_plural)
+        return self.cldr_plurals_list().index(cldr_id)
+
+    def get_cldr_plural(self, plural_form):
```

Comment:
Maybe `get_relative_cldr_plural` to make it more obvious how it's different from `cldr_id_to_plural`? And `plural_id` instead of `plural_form`.

Comment would be nice.